pandas/_libs/tslibs/resolution.pyx

```diff
@@ -373,21 +373,27 @@ cdef unique_deltas(ndarray[int64_t] arr):
     return result
 
 
-def _is_multiple(us, mult):
+cdef inline bint _is_multiple(int64_t us, int64_t mult):
     return us % mult == 0
 
 
-def _maybe_add_count(base, count):
+cdef inline str _maybe_add_count(str base, count):
```

Comment:
Can we add a int64_t type for `count` here?

Reply:
Ill double-check, but I'm pretty sure past-me had a reason for leaving that one out.  Maybe there are cases when `None` is passed